### PR TITLE
add FTMS support

### LINKED ISSUE #4: Add FTMS support
I don't have time myself to implement this (nor do I have an FTMS compatible trainer to test with), but if anyone would like to give it a go, pull requests would be welcome!

The specification can be found here: https://www.bluetooth.com/specifications/specs/

### PR DESCRIPTION
**Pull request by tensorturtle**
Fix #4 This commit implements all 7 characteristics of Fitness Machine Service (FTMS) and demonstrates resistance and power-based control, as well as printing various read/notify characteristics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +30,7 @@ Battery Service (BAS)| | ✓ |
 Cycling Speed and Cadence Service (CSCS) | | ✓ |
 Cycling Power Service (CPS) | | ✓ |
 Elite Sterzo Steering Service (STERZO)| | ✓ |
-FiTness Machine Service (FTMS) | | |✓
+FiTness Machine Service (FTMS) | | ✓ |
```
Comment (zacharyedwardbull) [inline README.md]: I think we can actually remove the "Not supported" column altogether. I added it originally to emphasise the fact FTMS isn't supported yet.

File: pycycling/sterzo.py [hunk 2]
```diff
@@ -6,7 +6,6 @@
 sterzo_control_point_id = '347b0031-7635-408b-8918-8ff3949ce592'
 sterzo_challenge_code_id = '347b0032-7635-408b-8918-8ff3949ce592'
 
-
```
Comment (zacharyedwardbull) [inline pycycling/sterzo.py]: Can we revert the change to this file, since the aim of the PR is to add FTMS

**Review by zacharyedwardbull — COMMENTED** [review]
LGTM. 

I think the only major change I'd like is using `namedtuples` rather than `dict` for all the return objects - this would make the API more consistent with the other classes in the library. `namedtuples` have the advantage of being immutable, and offer a slightly nicer way of accessing the values.

I've also left a few small comments.

**Comment by tensorturtle** [comment]
Hi @zacharyedwardbull thank you for the detailed comments. I have applied all the changes, please take a look.

**Comment by zacharyedwardbull** [comment]
Thanks, I'll have another look at the weekend!

**Comment by zacharyedwardbull** [comment]
Merged, thanks very much for the contribution!

**Comment by tensorturtle** [comment]
My pleasure! Random question - on the README, it says all of the profiles are 'partially supported'. What do you mean by 'fully supported'?

**Comment by zacharyedwardbull** [comment]
> Random question - on the README, it says all of the profiles are 'partially supported'. What do you mean by 'fully supported'?

I haven't implemented every feature of the protocols. Using the cycling power service as an example, the library doesn't yet support writing to the control point.
